roles/backup/tasks/postgres.yml

```diff
@@ -82,6 +82,15 @@
     resolvable_db_host: '{{ (awx_postgres_type == "managed") | ternary(awx_postgres_host + "." + ansible_operator_meta.namespace + ".svc.cluster.local", awx_postgres_host) }}'  # yamllint disable-line rule:line-length
   no_log: "{{ no_log }}"
 
+- name: Precreate database partitions
+  k8s_exec:
+    namespace: "{{ ansible_operator_meta.namespace }}"
+    pod: "{{ awx_task_pod_name }}"
```

Comment:
undefined variable `awx_task_pod_name`

Reply:
so you probably need a task in front of this to find awx_task_pod_name 

that takes `deployment_name` to find the awx resource than use that to find the name of a task pod for that deployment

Reply:
We need to query the k8s cluster for the name of the running awx pod to set that variable.  

Here is a good example of that:
* https://github.com/ansible/awx-operator/blob/devel/roles/installer/tasks/resources_configuration.yml#L2-L25